static/src/javascripts/projects/common/views/membership-message.html

```diff
@@ -6,5 +6,5 @@
             <span class="membership__support-button"><a class="message-button-rounded__cta <%=colourClass%>" href="<%=linkHref%>"><%=buttonCaption%><%=arrowWhiteRight%></a></span>
         </span>
     </div>
-    <a class="u-faux-block-link__overlay js-engagement-message-link" target="_blank" href="<%=linkHref%>" data-link-name="Read more link"></a>
+    <a class="js-engagement-message-link" target="_blank" href="<%=linkHref%>" data-link-name="Read more link"></a>
```

Comment:
I believe this change will break the (already broken) "do not adblock" message banner